### ISSUE TITLE
Recaptcha error on profile creation, running locally

### ISSUE THREAD

**Issue by BradHN1** [massenergize/frontend-portal#760]
![image (3)](https://user-images.githubusercontent.com/12800484/141645475-963ec8a4-adca-4e51-befe-98c9331a2884.png)

**Comment by BradHN1** [comment]
@dim505 tried to create a profile on the Dev portal, and got the same error.  
Following instructions on stackoverflow, I went on google recaptcha admin site https://www.google.com/recaptcha/admin/

i added recaptcha for the domain massenergize.dev (I hope that was the right thing to do) and got the following response:
![Screen Shot 2021-11-22 at 11 36 28 AM](https://user-images.githubusercontent.com/12800484/142901029-1ce1ea5c-b456-4cd7-9621-39df08d080db.png)

Looking at the frontend portal code, RegisterForm.js, the code has hardcoded the recaptcha key presumably for massenergize.org domain:
![Screen Shot 2021-11-22 at 11 45 52 AM](https://user-images.githubusercontent.com/12800484/142901376-2507e3cd-1b5b-4451-b150-0edfb2d138a6.png)

Comments I added are the site key and secret key for the massenergize.dev domain.
Question is - should these be kept secret to avoid a security hole?
